client/src/sockets/helpers/index.js

```diff
@@ -0,0 +1,48 @@
+import { ListEvents } from 'sockets/enums';
+import history from 'common/utils/history';
+
+export const listEventsController = (event, data, dispatch) => {
+  switch (event) {
+    case ListEvents.LEAVE_ON_TYPE_CHANGE_SUCCESS: {
+      const { cohortId, isCohortMember, listId } = data;
+
+      dispatch({ type: ListEvents.DELETE_SUCCESS, payload: listId });
+
+      return history.replace(
+        `/${isCohortMember ? `cohort/${cohortId}` : 'dashboard'}`
+      );
+    }
+    case ListEvents.REMOVE_MEMBER_SUCCESS: {
+      const { cohortId } = data;
+
+      if (cohortId) {
+        dispatch({ type: event, payload: data });
+        history.replace(`cohort/${cohortId}`);
+
+        return window.location.reload();
```

Comment:
hmm, if this is good for the user to reload to the window if someone else removes them from the list? What about notification: why the user is redirected?